tests/unit/test_main.py

```diff
@@ -2565,3 +2565,41 @@ def test_check_no_tag_duplication(self):
 
         self.napp._check_no_tag_duplication(evc_id, None, None)
         assert evc.check_no_tag_duplicate.call_count == 3
+
+    @patch("napps.kytos.mef_eline.main.settings")
+    @patch("napps.kytos.mef_eline.main.Main.handle_interface_link_up")
+    @patch("napps.kytos.mef_eline.main.Main.handle_interface_link_down")
+    def test_handle_on_interface_link_change(
+        self,
+        mock_down,
+        mock_up, 
+        mock_settings
+    ):
+        """Test handle_on_interface_link_change"""
```

Comment:
- Let's mock time.sleep too
- Also for completeness let's cover an unordered KytosEvent being called

Reply:
`time.sleep` mocked. `settings` was not necessary
Unordered events also covered.